server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -2966,13 +2966,26 @@ public Void run() {
     }
   }
 
+  private Durability getMincEventDurability(KeyExtent extent) {
+    TableConfiguration conf;
+    if (extent.isMeta() || extent.isRootTablet()) {
```

Comment:
> Is there some other setting for user tables that's more appropriate? 

This is the best setting I know of... Since the walog minc entries are used during recovery, its nice to have them at the same sync level as the tablets metadata.

> so the || extent.isRootTablet() is redundant

Nice catch, I wrongly assumed isMeta() was a more narrow check.  I used to know it did a wider check, but had forgotten.  

>  I think refactoring KeyExtent#isMeta is more appropriate

Changing its name would need to be a separate PR, because it would be very noisy drowning out the changes for 4112.  I agree another name that implies more than `accumulo.metadata` would be good, however I am not sure what that name would be.